src/coord/src/catalog/builtin.rs

```diff
@@ -1145,6 +1145,18 @@ pub static MZ_CLUSTER_REPLICAS: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
         .with_column("availability_zone", ScalarType::String.nullable(true)),
 });
 
+pub static MZ_AUDIT_LOG: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
+    name: "mz_audit_log",
```

Comment:
I know these column names match the design doc, but I have same late in the game suggestions to make them more consistent with the rest of the system catalog!

```suggestion
    name: "mz_audit_events",
```

(Most tables are named after the plural form of what they contain. I propose we call these "audit events"/"events" to match what the code calls them. Two reasons to prefer "events" to "operations", IMO. First is that it is shorter. Second is that we may eventually want to audit things like reads which I are less clearly "operations".)